component-samples/demo/db/docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+# Copyright 2020 Intel Corporation
+# SPDX-License-Identifier: Apache 2.0
+
+# Use root/example as user/password credentials
+version: '3.1'
+
+services:
+
+  fdo-db:
+    image: mariadb
```

Comment:
Can we move this fdo-db container to root `docker-compose.yml` file ? So that both services can start at the same time. We used to follow that approach in SDO RV -- Redis and SDO RV service in same docker-compose.yml file.